blueskyacademy/bluesky-nextjs
# add testimonial parents

## Description
- User can see feedback from parents in Parent Page

## Why
- User can see a list of feedback in Parent Page as Slider

## How to QA
- Admin can add feedback from Contentful (English & Vietnamese)
- User can see feedback in HomePage and Parent Page

## Screenshots
![Screen Shot 2022-01-19 at 11 04 10](https://user-images.githubusercontent.com/74655110/150061931-7a45072a-9d08-4582-a935-c41c4dbf749e.png)
